deploy/crds/marketplace.redhat.com_v1alpha1_marketplaceconfig_cr.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: marketplace.redhat.com/v1alpha1
 kind: MarketplaceConfig
 metadata:
-  name: rhm-marketplaceconfig
+  name: marketplaceconfig
```

Comment:
This shouldn't be changed - it's ok that it's the wrong name. The problem is that the operator scanning fails if it's the name we use for standard deployment. If it's this name, the scan breaks.
```suggestion
  # NOTE: Named this way to prevent an error in scanning
  name: rhm-marketplaceconfig
```